### ISSUE TITLE
Hardware Probe not working

### ISSUE THREAD

**Issue by OspreyJr** [helloSystem/Utilities#152]
In helloSystem 0.8.1 Build 290 13.1 release, live booting from usb drive, Lenovo T520

Button "show hardware probe" has no reaction on clicking (screenshot).
Pressing button "continue" two times brings warning (screenshot 2).

![screenshot](https://user-images.githubusercontent.com/112200118/219876349-63f051d5-9310-493a-87a3-d695ec9f0663.jpg)
![screenshot2](https://user-images.githubusercontent.com/112200118/219876357-4468ffaa-d06d-482b-b86f-5136fc2a88c9.jpg)

**Comment by probonopd** [comment]
Please paste the output of

```
env | grep LANG
```

Mine is:

```
LANG=de_DE.UTF-8
LANGUAGE=
```

**Comment by OspreyJr** [comment]
Mine is unchanged from the live iso:
`
LANG=en

LANGUAGE=
`

**Comment by probonopd** [comment]
This seems to be dependent on the language, as I don't have this issue on my system which is set to German using https://github.com/helloSystem/hello/wiki/EFI-NVRAM:

![VirtualBox_helloSystem_18_02_2023_20_04_04](https://user-images.githubusercontent.com/2480569/219883664-5c4b7a1b-69f5-48e8-9b86-b6d1c7d03ba1.png)

**Comment by probonopd** [comment]
I can confirm that uploading fails if `LANG=en`.

**Comment by probonopd** [comment]
Workaround: Open QTerminal, run:

```
LANG=en_US.UTF-8
launch "Hardware Probe"
```

**Comment by OspreyJr** [comment]
Yes, that helped regarding uploading. When pressing the Show Hardware Probe button there is still no reaction besides that output in Terminal: 

showHardwareProbeButtonClicked
self.local_probe_path: /root/HW_PROBE/LATEST/hw.info
Starting launch ['Filer', '/root/HW_PROBE/LATEST/hw.info']

**Comment by probonopd** [comment]
What happens if you open Hardware Probe, and then instead of clicking on the button enter in QTerminal:

```
launch Filer '/root/HW_PROBE/LATEST/hw.info'
```

**Comment by OspreyJr** [comment]
Then a second instance of Hardware Probe opens, also not reacting Show Hardware Probe button.

**Comment by probonopd** [comment]
What, `launch Filer ...` opens Hardware Probe? Something must be seriously wrong.

**Comment by OspreyJr** [comment]
Sorry, maybe my mistake! Copy and paste is a bit messy with Falkon... it opens a window of Filer (hw.info) showing three icons: devices, host and logs folder.

**Comment by probonopd** [comment]
And when you click the button it doesn't do that? Even though it wants to execute that exact same command?

**Comment by OspreyJr** [comment]
Today I played around with the Hardware Probe a little bit. It seems (I didn't look at the code) that it collects the info upon starting, not when pressing any button. Pressing the Show Hardware Probe button didn't have any effect on the button nor on the files in hw.info. When I close Hardware Probe and start it again, the hw.info folder doesn't seem to xist
